lms/djangoapps/shoppingcart/models.py

```diff
@@ -1196,7 +1197,7 @@ def is_registration_code_redeemed(cls, course_reg_code):
         Checks the existence of the registration code
         in the RegistrationCodeRedemption
         """
-        return cls.objects.filter(registration_code=course_reg_code).exists()
+        return cls.objects.filter(registration_code__code=course_reg_code).exists()
```

Comment:
Thanks for the explanation. I see that you changed the parameter that you are passing in from the existing caller. Thanks.